package.json

```
@@ -15,8 +15,10 @@
     "setAssetMeta": "node scripts/setAssetMeta.js",
     "setAssetMetaViaJSON": "node scripts/setMetadataViaStorage.js",
     "startEthNode": "./scripts/startEthNode.sh",
+    "setInitialValidators": "hardhat run scripts/forceActiveValidators.js --network $1",
     "validatorRelayer": "hardhat run scripts/subscribeEventProof.js --network $1",
     "claimRelayer": "hardhat run scripts/subscribeEthereumDeposit.js --network $1",
+    "publish": "npx hardhat verify --network ropsten $1",
```

```suggestion
    "publish": "npx hardhat verify --network $1 $2",
```

`yarn publish ropsten ...`